### DISCUSSION TITLE
How much time will take a Ryzen 3900x to do a 2hs movie at x265?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [staxrip/staxrip#442] [Q&A] [answered]
At this settings:

- Quality = **18**
- Preset = **Medium**

Thx in advance.

**Comment by Dendraspis** [comment, marked as answer]
1080p or 2160p? Cropped or not?

Totally depends on the source. Also which CRF value you should use.
An 2160p animation movie could be done in about 6h, a more grainy movie in about 10h I guess - have never tried ***Medium*** preset.

But I suggest you do your own test with a source that fits your average source files in terms of quality.  ☺️

**Reply by ZeroCool22** [reply]
- **1080p.**
- **No Crop.**

I can't do a test because i don't have the CPU yet. :D

**Reply by Dendraspis** [reply]
🤣 

Ok, so I would guess 3h for a flat animation movie and about 4h for a normal movie. But as I said this depends totally on the source and I have never used ***Medium*** preset.

**Reply by ZeroCool22** [reply]
What preset do you use, slow?

**Reply by Dendraspis** [reply]
@ZeroCool22 
Please stop quoting the ***last*** comment - this kills the overview in longer discussions.  😉 

Of course - this is where the magic begins. Even slower presets are a little bit better, but they need a lot more time, so *Slow* is a good (the best for many users) compromise between time, quality and file size.

You really should take a difficult source with fast movement, dark scenes and a good quality and run your own tests. Create a 10min sample and find your personal best compromise.

**Comment by ZeroCool22** [comment]
I guess your calculations are right, because with a 7700k it takes:

**Slow**
![Slow](https://user-images.githubusercontent.com/13344308/104459269-46b4c800-558b-11eb-99c5-0828462f10b1.png)

**Medium**
![Medium](https://user-images.githubusercontent.com/13344308/104459273-474d5e80-558b-11eb-952e-8879791026a9.png)

Damn, i tough a 3900x would be a little more fast... 😬

I guess with this processor, i will keep with GPU encoding all the way... 😁

**Reply by Dendraspis** [reply]
Estimated time is useless for such an comparison, especially if you compare one encode at 653 and the other at 1220 frames. ETA might raise or fall a lot at the beginning. Do a full run with same settings and compare. A Ryzen 3900X should be about 2.5x faster.

> I guess with this processor, i will keep with GPU encoding all the way... 😁

When it makes you happy...  😁

**Comment by ghost** [comment]
Keep in mind that most Ryzen users have reported the CPU usage around or below 50%. In order to fully take advantage of you Ryzen CPU, either you encode one film using 2 or 3 chunks, or encode two films at the same time (with two Staxrip instances running at the same time).

**Reply by ZeroCool22** [reply]
The solution for that: https://www.youtube.com/watch?v=BU8nkD9GZsg

**Reply by ghost** [reply]
This is a good but now "old" video stressing on the need to select Wavefront Parallel processing, which is now selected by default !!
You can still tamper with the "Threads" value, but I'm afraid the hint that he is showing won't help you now...

What you must know: 

- If you are encoding 4K, you'll get 100% CPU all right.
- If you're encoding FHD, then you'll not get 100% CPU and the solution for you in that case is to use the "Chunks" feature in x265 options > Input / Output > "Chunks", set it to 2 or 3.
- If you're encoding HD or less, then you'll need to set the Chunks to 4 or 5 and need to increase the max number of parallel processes to 5 in the settings > System (and leave it to 5 afterwards)
